elements/pl-dropdown/pl-dropdown.py

```diff
@@ -0,0 +1,134 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+from enum import Enum
+
+WEIGHT_DEFAULT = 1
+
+
+class SortTypes(Enum):
+    RANDOM = 'random'
+    ASCEND = 'ascend'
+    DESCEND = 'descend'
+
+
+def parse_separated_list(raw_str):
+    return [opt.strip() for opt in raw_str.splt(',')]
```

Comment:
```suggestion
    return [opt.strip() for opt in raw_str.split(',')]
```

Reply:
Thanks! Thought I fixed this but I missed it. Fixed :)